packages/components/src/theme/index.scss

```diff
@@ -28,7 +28,7 @@
 @import "grid";
 @import "cards";
 @import "footer";
-@import "labelHelpTip";
```

Comment:
are you sure you wanted to remove this `labelHelpTip.scss` file from the index.scss?

Reply:
Yep! This was actually a duplicate line. Should have called that out for this change. We can see "labelHelpTip" is imported on line 19 still.

Reply:
Ah yes, Thanks.